docs/tutorial/source/tutorial-2.py

```diff
@@ -0,0 +1,114 @@
+import toga
```

Comment:
Is this how it's supposed to be? This doesn't seem to match the tutorial doc below.

Reply:
This is copy-pasta from the Toga tutorial.